packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -430,6 +430,75 @@ abstract class ScrollView extends StatelessWidget {
 /// ```
 /// {@end-tool}
 ///
+/// {@tool dartpad --template=stateful_widget_material}
+///
+/// [ListView] is often used to display items in a scrolling widget. If items
```

Comment:
Here's a word-smithed version, see what you think:

By default, if items are inserted at the "top" of a scrolling container like [ListView] or [CustomScrollView], the top item and all of the items below it are scrolled downwards.  In some applications, it's preferable to have the top of the list just grow upwards, without changing the scroll position. This example demonstrates how to do that with a [CustomScrollView] with two [SliverList] children. The top one SliverList will grow upwards, and the bottom SliverList will grow downwards.

Reply:
I like this! That's better. 

I only changed one sentence to mention the center key part:
```This example demonstrates how to do that with a [CustomScrollView] with two [SliverList] children, and the [CustomScrollView.center] set to the key of the bottom SliverList.```